src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -313,7 +313,12 @@ def get_api_version(self, resource_type=None, operation_group=None):
         resource_type = resource_type or self.command_kwargs.get('resource_type', None)
         return self.loader.get_api_version(resource_type=resource_type, operation_group=operation_group)
 
-    def supported_api_version(self, resource_type=None, min_api=None, max_api=None, operation_group=None):
+    def supported_api_version(self, resource_type=None, min_api=None, max_api=None,
+                              operation_group=None, parameter_name=None):
```

Comment:
I don't think it is intuitive to insert `parameter_name` to `supported_api_version`. How about creating another function `supported_parameter` and call `supported_api_version` internally?